src/components/data-display/tables/UndernamesTable.tsx

```diff
@@ -467,41 +458,33 @@ const UndernamesTable = ({
             });
             setAction(undefined);
           }}
-          antId={antId}
+          antId={arnsRecord.processId}
+        />
+      )}
+      {action === UNDERNAME_TABLE_ACTIONS.EDIT && selectedUndername && (
```

Comment:
```suggestion
      {action === UNDERNAME_TABLE_ACTIONS.EDIT && selectedUndername ? (
```

see comment above - use ternary operators instead of `&&`